src/components/molecules/AlarmHistoryItem.tsx

```diff
@@ -3,18 +3,20 @@ import styles from "@/styles/components/molecules/AlarmHistoryItem.module.scss";
 
 type Props = {
   data: {
+    is_alarm: boolean;
```

Comment:
Are you using this `is_alarm`?

Reply:
It's not used but it is being sent from the server, so probably need to include for TS?

Reply:
It's not being rendered but may be useful to check only true items are being passed in - we can strip it out if you feel that's cleaner just requires a little extra code in the parent component

Reply:
Have removed as we will move the filtering logic to backend